parm/archive/enkf.yaml.j2

```diff
@@ -118,3 +118,8 @@ enkf:
 
         {% endif %}  # End IAU
         # End of analysis mean increments/analyses
+
+        {% if DOHYBVAR_OCN %}
+        - "{{ COMIN_OCEAN_ANALYSIS_ENSSTAT | relpath(ROTDIR) }}/{{ head_ocean }}bg_ensvar.nc"
+        - "{{ COMIN_ICE_ANALYSIS_ENSSTAT | relpath(ROTDIR) }}/{{ head_ice }}bg_ensvar.nc"
```

Comment:
Do you know which jobs produce these files?

Reply:
`ocnanalecen` - Are you fixing the dependency issue?

Reply:
Yes, but I see now that no dependency updates are required.  It's just the updates to enkf_marine_grp.yaml.j2 and exgdas_enkf_earc_tars.py.  I'm testing these changes now.  Do you mind if I open a PR into your branch when I'm done?

Reply:
Actually, since this comes with a GDASApp update, I think I will create a separate PR into develop.